<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabServer">
       <attribute name="title">
        <string>Server</string>
       </attribute>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>491</width>
          <height>91</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEditPort">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>10</y>
           <width>211</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonListen">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>40</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Listen</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonListenClose">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>40</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Closed</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_5">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>10</y>
           <width>21</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>In</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_6">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>40</y>
           <width>21</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Out</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>10</y>
           <width>181</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>Image TBD</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tabClient">
       <attribute name="title">
        <string>Client</string>
       </attribute>
       <widget class="QFrame" name="frame_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>491</width>
          <height>91</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Host:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEditHostClient">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>10</y>
           <width>151</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEditPortClient">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>40</y>
           <width>151</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>10</y>
           <width>21</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_7">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>40</y>
           <width>21</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonConnectClient">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>10</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonCloseClient">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>40</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Closed</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_9">
         <property name="geometry">
          <rect>
           <x>340</x>
           <y>10</y>
           <width>21</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>In</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_10">
         <property name="geometry">
          <rect>
           <x>340</x>
           <y>40</y>
           <width>21</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Out</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>380</x>
           <y>10</y>
           <width>51</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>Image TBD</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
